rero_ils/modules/loans/cli.py

```diff
@@ -245,6 +245,10 @@ def create_loan(barcode, transaction_type, loanable_items, verbose=False,
                 requested_patron.patron_type_pid,
                 item.item_type_pid
             )
+            if requested_patron.is_blocked:
```

Comment:
I do not understand why you are updating the `cipo` record.

Reply:
It was a test to check if it has an impact. It would be useful to pass a context or something like that. But I don't know how yet.